packages/@webex/plugin-meetings/src/member/types.ts

```diff
@@ -23,6 +23,14 @@ export type ParticipantWithRoles = {
   };
 };
 
+export type ParticipantWithBrb = {
+  controls?: {
```

Comment:
One thing that I haven't found easily is in what situations `controls` can be `undefined`. I would appreciate it if someone could highlight that. For now, I will leave this part consistent with other member logic and add optional `controls` logic.